src/Analyzers/VisualBasic/Tests/SimplifyObjectCreation/SimplifyObjectCreationTests.vb

```diff
@@ -0,0 +1,206 @@
+﻿' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports Microsoft.CodeAnalysis.VisualBasic.CodeStyle
+Imports VerifyVB = Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions.VisualBasicCodeFixVerifier(
+    Of Microsoft.CodeAnalysis.VisualBasic.SimplifyObjectCreation.VisualBasicSimplifyObjectCreationDiagnosticAnalyzer,
+    Microsoft.CodeAnalysis.VisualBasic.SimplifyObjectCreation.VisualBasicSimplifyObjectCreationCodeFixProvider)
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.SimplifyObjectCreation
+    Public Class SimplifyObjectCreationTests
+        <Fact>
+        Public Async Function SimplifyObjectCreation() As Task
```

Comment:
is there a fix-all test?

Reply:
@CyrusNajmabadi See `SimplifyObjectCreation_MultipleDeclarators`.